components/scripts/gradle/01-validate-incremental-building.sh

```diff
@@ -22,9 +22,9 @@ readonly LIB_DIR="${SCRIPT_DIR}/lib"
 
 # Include and parse the command line arguments
 # shellcheck source=lib/01-cli-parser.sh
-source "${LIB_DIR}/${EXP_NO}-cli-parser.sh" || { echo "Couldn't find '${LIB_DIR}/${EXP_NO}-cli-parser.sh' library."; exit 1; }
+source "${LIB_DIR}/${EXP_NO}-cli-parser.sh" || { echo -e "\033[00;31m\033[1mERROR: Couldn't find '${LIB_DIR}/${EXP_NO}-cli-parser.sh' parsing library.\033[0m"; exit 2; }
```

Comment:
The funny looking `\033[00;` bits are to change the text color to red, bold the text, and then reset the formatting to the terminal's default after the error message has been printed.

Elsewhere we hide this with functions and constants, but since we haven't loaded those functions and constants yet, we can't rely on them here. Same reason why the exit code is hardcoded rather than using one of our constants.